.github/workflows/check-public-apis.yml

```diff
@@ -0,0 +1,76 @@
+name: Check for new public APIs
+on:
+  pull_request:
+    types: [opened, labeled, unlabeled, synchronize]
+    paths:
+      - '**/*.swift'
+      - '!StripeFinancialConnections/**'
+      - '!StripeIdentity/**'
+      - '!StripeConnections/**'
+    branches-ignore:
+      - 'releases/*' # ignore release PRs due to them modifying StripeAPIConfiguration.STPSDKVersion
```

Comment:
Do we need to ignore:
```
docs*
```
?

Reply:
Good callout! I didn't consider this case actually, but since the docs PR only modifies `.json` files this workflow shouldn't be trigged since it looks for `.swift` file changes!

Reply:
If no swift files are modified the workflow just won't be triggered! See on this PR how the latest check it's not ran since no swift files are currently modified. If you tap into the checks on earlier commits you'll see it being ran.